tango_simlib/tests/test_fandango_json_parser.py

```diff
@@ -89,12 +89,12 @@ class GenericSetup(unittest.TestCase):
     def setUp(self):
         super(GenericSetup, self).setUp()
         self.fandango_json_file = pkg_resources.resource_filename(
-            'tango_simlib.tests.config_files', 'database2.json')
+            'tango_simlib.tests.config_files', 'database2.fgo')
         self.fandango_parser = fandango_json_parser.FandangoExportDeviceParser()
         self.fandango_parser.parse(self.fandango_json_file)
 
 
-class TestFandangoJsonParser(GenericSetup):
+class test_FandangoJsonParser(GenericSetup):
```

Comment:
For interest sake, why the change to `test_` ?

Reply:
Oh it's on page 15 of the CAM - Python style guide for naming a test class.